include/battle_ai_util.h

```diff
@@ -4,13 +4,14 @@
 #define FOE(battler) ((BATTLE_OPPOSITE(battler)) & BIT_SIDE)
 
 // Roll boundaries used by AI when scoring. Doesn't affect actual damage dealt.
-#define MAX_ROLL_PERCENTAGE DMG_ROLL_PERCENT_HI 
+#define MAX_ROLL_PERCENTAGE DMG_ROLL_PERCENT_HI
 #define MIN_ROLL_PERCENTAGE DMG_ROLL_PERCENT_LO
+#define DMG_ROLL_PERCENTAGE (MAX_ROLL_PERCENTAGE - 7) // Controls the damage roll the AI sees. By default the 9th roll is seen
```

Comment:
```suggestion
#define DMG_ROLL_PERCENTAGE ((MAX_ROLL_PERCENTAGE + MIN_ROLL_PERCENTAGE) / 2 + (MAX_ROLL_PERCENTAGE + MIN_ROLL_PERCENTAGE ) % 2) // Controls the damage roll the AI sees. By default the 9th roll is seen
```

Reply:
hmm. Isn't this convoluted just for the sake of it?

Reply:
I can see this being a good catch if the user sets say, the min percentage to be 85 and the max percentage to be 90. This'll make DMG_ROLL_PERCENTAGE show a number that isn't within the specified range because of the flat -7

Reply:
ah I see. This makes sense, yeah

Reply:
Yeah it's just `(MAX_ROLL_PERCENTAGE + MIN_ROLL_PERCENTAGE + 1) / 2`